src/https:/googleapis/github/io/google-cloudevents/jsonschema/google/events/firebase/remoteconfig/v1/RemoteConfigEventData/json/RemoteConfigEventData.py

```diff
@@ -0,0 +1,46 @@
+from typing import Optional, Union
```

Comment:
Do we know what happened to the header here?

Reply:
Sorry I am not sure I understand the question here. By header you mean the copyright disclaimer?

Reply:
Yes, like the sister `__init__.py` file. The `Copyright 2020 Google LLC...` header. Would be nice to have.

Reply:
Done.